hydra-nextjs/tests/helpers.js

```diff
@@ -0,0 +1,18 @@
+import http from 'http';
+
+export function mockRequest(options) {
+  let req = new http.IncomingMessage();
+
+  const { body, ...attrs } = options || {};
+  req = Object.assign(req, attrs);
+
+  // write body
+  req.push(body || '');
+  req.push(null);
```

Comment:
Does pushing `null` do something useful here?

Reply:
Pushing `null` ends the `IncomingMessage`, however I think I'll switch to the node-mocks-http library which will replace this custom code